5.2.join.py

```diff
@@ -0,0 +1,18 @@
+def join(*params, sep = '-'):
+    if not params:
+        return None
+    L = []
```

Comment:
Rename 'L' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing